plugin.audio.intergalacticfm/main.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+# Module: default
+# Author: Pander
+# Created on: 2019-03-04
+# License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+
+from json import load
+from sys import argv
+from os.path import isfile
+import requests
+
+from urllib import urlencode
+from urlparse import parse_qsl
+
+
+import xbmcgui
+import xbmcplugin
+import xbmcaddon
+from xbmc import log, LOGNOTICE, LOGERROR
+
+__addonid__ = "plugin.audio.intergalacticfm"
+base = xbmc.translatePath('special://home/addons/{}/resources/'.format(__addonid__))
+
+# Get the plugin url in plugin:// notation.
+_url = argv[0]
+# Get the plugin handle as an integer number.
+_handle = int(argv[1])
+
+_addon = xbmcaddon.Addon()
+
+
+radio = 'http://radio.intergalactic.fm:80/'
+
+streams = load(open(base + 'streams.json'))
+
+def list_audios():
+    """
+    Create the list of playable streams in the Kodi interface.
+    """
+
+    listing = []
+
+    xbmcplugin.setPluginCategory(_handle, 'Live Streams')
+    xbmcplugin.setContent(_handle, 'videos')
+
+    for key, audio in streams.items():
+        list_item = xbmcgui.ListItem(label=audio['label'])
+        list_item.setInfo(type='video', infoLabels={'genre': audio['genre'], 'plot': audio['plot'], 'tagline': audio['tagline']})
+
+        # see https://kodi.wiki/view/Movie_artwork
+        # only poster, fanart and clearlogo is supported/needed
+
+        art = {}
+
+        #log(__addonid__ + ' key: ' + key, LOGNOTICE)
+        # poster 1000x1500 1:1.5 PNG
+        poster = '{}{}-poster.png'.format(base, key)
+        if isfile(poster):
+            art['poster'] = poster
+        else: # note: specific fallback
+            art['poster'] = '{}intergalactic_radio-poster.png'.format(base)
+        #log(__addonid__ + ' poster: ' + art['poster'], LOGNOTICE)
+
+        # fanart 1920x1080 16:9 JPG
+        fanart = '{}{}-fanart.jpg'.format(base, key.split('_')[0])
+        if isfile(fanart):
+            art['fanart'] = fanart
+        else: # note: specific fallback
+            art['fanart'] = '{}fanart.jpg'.format(base)
+        #log(__addonid__ + ' fanart: ' + art['fanart'], LOGNOTICE)
+
+        # clearlogo 800x310 1:0.388 transparent PNG (is top-left corner overlay)
+        art['clearlogo'] = '{}intergalactic_radio-clearlogo.png'.format(base)
+
+        list_item.setArt(art)
+        list_item.setProperty('IsPlayable', 'true')
+
+        url = '{}{}'.format(radio, audio['file'])
+        log(__addonid__ + ' url: ' + url, LOGNOTICE)
```

Comment:
You need to use debug logging (see https://kodi.wiki/view/Add-on_rules)

Reply:
done